deploy/runtime/defaults/PVWatts Wind Battery Hybrid_Host Developer.json

```diff
@@ -55917,7 +55918,7 @@
         "ppa_escalation": 1.0,
         "ppa_multiplier_model": 0.0,
         "ppa_price_input": [
-            0.187
+            0.153
```

Comment:
General comment/question: solar resource is in Phoenix while wind resource is in Wyoming. Do we want to do something with that or just keep the existing defaults from the individual technologies?